packages/react/focus-scope/src/FocusScope.tsx

```diff
@@ -21,10 +21,16 @@ type FocusScopeOwnProps = Polymorphic.Merge<
   Polymorphic.OwnProps<typeof Primitive>,
   {
     /**
-     * Whether focus should be trapped within the FocusScope
+     * Whether focus should wrap around the `FocusScope`.
      * (default: false)
      */
-    trapped?: boolean;
+    wrapped?: boolean;
+
+    /**
+     * Whether focus should be contained inside the `FocusScope`.
+     * (default: false)
+     */
+    contained?: boolean;
```

Comment:
Is it worth being more descriptive on the difference?

Reply:
Will do.